<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AvgLumEditWidget</class>
 <widget class="QWidget" name="AvgLumEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>42</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="ICEAVGLUMPROC_NS::TopItemWidget" name="topItemWidget" native="true"/>
   </item>
   <item>
    <widget class="ICEAVGLUMPROC_NS::ItemWidget" name="item1Widget" native="true"/>
   </item>
   <item>
    <widget class="ICEAVGLUMPROC_NS::ItemWidget" name="item2Widget" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ICEAVGLUMPROC_NS::TopItemWidget</class>
   <extends>QWidget</extends>
   <header>topitemwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ICEAVGLUMPROC_NS::ItemWidget</class>
   <extends>QWidget</extends>
   <header>itemwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../iceavglumproc.qrc"/>
 </resources>
 <connections/>
</ui>
